test/integration/namespace-test.js

```diff
@@ -35,5 +35,5 @@ test('shutdown sandbox', async t=> {
 process.on('unhandledRejection', (reason, p) => {
   console.log(reason)
   console.log(p)
-  end()
+  sandbox.end()
```

Comment:
This doesn't work. Sandbox keeps running, but the code that was previously here was using the outdated `end()` syntax that threw an unrelated error when an unhandled promise rejection happened.